localization:

  # Stop ICP if the transformation from the last iteration was this small.
  tf_epsilon: 0.0000000001

  # During ICP, two points won't be considered a correspondence if they are
  # at least this far from one another.
  corr_dist: 1.0

  # Iterate ICP this many times.
  iterations: 10

  # Maximum acceptable incremental rotation and translation.
  transform_thresholding: false
  max_translation: 0.05
  max_rotation: 0.1
